app/views/events/show.html.erb

```diff
@@ -1,69 +1,70 @@
-<div class="container-fluid" style="padding-top: 20px; padding-left: 30px">
+<!-- Event title -->
+<div class="jumbotron">
+  <div class="container-fluid">
+    <h1 class="display-4" style="padding-left: 1%"><%= @event.name%></h1>
+    <br>
+    <h3 style="padding-left: 1%">
+      <strong>When:</strong>
```

Comment:
![image](https://user-images.githubusercontent.com/20048451/57557393-d45c0100-732e-11e9-9760-da4affe5a045.png)

Currently this is not how the footer should behave. The header (not the navbar, the thing with the back button in it) looks much too large.